spec/support/view_component/context.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Support
+  module ViewComponent
+    class Context
+      StubbedComponent = Struct.new(:instance, :args, :output)
+
+      def register_component(instance, args)
+        components << StubbedComponent.new(
+          instance,
+          args,
+          Faker::Lorem.paragraphs.join(" "),
```

Comment:
Uhh, are we sure it's a good idea to have a bunch of lorem ipsum content in our stubbed components? Can we maybe do something like `Stubbed Component #{instance}`, or something to that effect? I don't know when this gets actually output, if at all.

Reply:
I'd like to enforce usage of our matchers instead of `expect(page).to have_content("Stubbed Component"), which is why I'd prefer random content. This content is being outputed and completely replaces the stubbed component output.

Reply:
Ah, that argument works for me. Random output reduced to uuids now. :)